### PR TITLE
Add full specification

### PR DESCRIPTION
**Pull request by joker314**
I don't know if it's useful, if it's not... feel free to close PR. :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,3 +5,52 @@
...
+
+You should not attempt to encode any other schema, as that will make the redirectatob invalid.
+
+### Encoding a URL
+
+#### In the Latin1 range
+
+If a destination URL contains NO characters OUTSIDE the Latin1 character range, then it should be encoded through the following steps
+
+1) Base64 encode the URL, in a way that replicates JavaScript's `btoa()` function.
```
Comment (towerofnix) [inline README.md]: This is just one step, so you may as well get rid of "through the following steps"; replace these two lines with:

>If a destination URL contains NO characters OUTSIDE the Latin1 character range, then it should be Base64-encoded, in a way that replicates JavaScript's `btoa()` function.

File: README.md [hunk 2]
```diff
@@ -5,3 +5,52 @@
...
+
+#### Outside the Latin1 range
+
+If a destination URL contains ONE OR MORE characters OUTSIDE the Latin1 range, then it should be encoded through the following steps...
```
Comment (towerofnix) [inline README.md]: Nit: This should be a colon, not an ellipsis.

File: README.md (hunk 1 above)
Comment (joker314) [reply README.md]: I think protocol refers to like "Now send this packet and do that"; whereas HTTP is the set of characters at the beginning of the URL which tell the browser which protocol to use. Okay, will fix -- I just like the two being symmetric

File: README.md (hunk 2 above)
Comment (joker314) [reply README.md]: True, I just don't like colons followed by newlines for some reason

**Comment by joker314** [comment]
Uh I _think_ I fixed the things

**Pull request merged**